CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ For details about compatibility between different releases, see the **Commitment
 
 - Increased the maximum gRPC message size to 16MB.
 - Gateways which have been deleted are now automatically disconnected by the Gateway Server.
+- Disabled devices that are registered on a different cluster.
```

Comment:
```suggestion
- Mark off and hide and restrict access to end devices that are registered on a different cluster in the Console.
```

Reply:
Avoid speaking of "devices" only since this can be ambiguous (could also mean gateways). Also, we typically mark Console-related changes with "in the Console" in the changelog.